src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/OverhangFixingManager.java

```diff
@@ -202,23 +231,28 @@ private void cleanSplices() {
      * @param read        the read to fix
      * @param splice      the split (bad region to clip out)
      */
-    private void fixSplit(final SplitRead read, final Splice splice) {
+    void fixSplit(final SplitRead read, final Splice splice) {
```

Comment:
I'd probably rename `read -> splitRead` and then bind `read.read` and `read.loc` to their own variables in this function for reduced verbosity and ease of understanding.  The logic this function is super confusing as is so anything that makes it easier to read is good.